### ISSUE TITLE
Support for different providers for password storage.

### ISSUE THREAD

**Issue by edmundlaugasson** [MeanEYE/Sunflower#40]
From another issue (https://github.com/MeanEYE/Sunflower/issues/38) came up an idea to file that feature as separate issue and here it is.
About LastPass - https://lastpass.com/
There is also KeePass - http://keepass.info/
The question is - which password manager(s) to support. LastPass seems to be most professional (unified look and feel on all platforms) and widely used and still free to use. Also it is integrated with XMarks bookmark manager.

Idea is to improve security and not store passwords locally as the encryption of local passwords might be weak and these might be easy to steal by malware.
But there might be cases like missing internet connection - then Gnome Keyring can be used but python-gnomekeyring must be installed prior using. This and other possible dependencies should be mentioned e.g. in documentation section.

EDIT: there could be similar plugin system like for web browsers currently are. Or adapt Sunflower to be able use some of the current web browsers plugin. If to create separate plugin system then probably creators (e.g. LastPass, KeePass, etc) would like also add such plugin to their web sites.

EDIT2: there could be optional choice, which password manager user would like to use.

**Comment by jtojnar** [comment]
KeepassXC has an [option](https://github.com/keepassxreboot/keepassxc/blob/88a0a8d35a19f9871e35191ff5ecd440e5fbcad4/CMakeLists.txt#L56) to expose Secret Service API so people will be able to use that once #439 is finished.
